CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### citus v10.0.1 (February 19, 2021) ###
+
+* Fixes an issue in creation of `citus.time_partitions` view
```

Comment:
```suggestion
* Fixes an issue in creation of `time_partitions` view
```

In the migration script we change the schema later on.

`ALTER VIEW citus.time_partitions SET SCHEMA pg_catalog;`

Reply:
hmm, I didn't realize that, thanks :)